SPEC.md

```diff
@@ -85,33 +85,54 @@ This specification outlines the requirements for botbuilder samples.
 ```markdown
 <INSERT AT MOST ONE PARAGRAPH DESCRIPTION OF WHAT THIS SAMPLE DOES> 
 
+# Table of Contents
+<LINKS TO SECTIONS OF THE README>
+
 # Concepts introduced in this sample
 <DESCRIPTION OF THE CONCEPTS>
 
+#Services used in this sample
+<INTRODUCE SERVICES>
```

Comment:
This is apparently ambiguous. I'd enumerate each service used in the sample, including its type (LUIS | QnA Maker | Dispatch), its name as it will appear in the **.bot** file, and a brief description of the service (as opposed to the type of service). Perhaps include in the template here, a boilerplate description for each type of service.

Reply:
Good suggestion, but this is just the spec, not a template.  I'll move on to a template after we all agree on this.